### PR TITLE
Quick attempt to add more helpful information to the README.

### PR DESCRIPTION
**Pull request by ketch**
I just noted down things that I had to figure out.  Most of this is specific to latex conversion.  Feel free to modify.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -27,3 +19,59 @@ To install locally as an editable install, run::
...
+(in markdown) the section heading ``## bar`` within the notebook starting with top-level
+header ``# foo``, then the markdown hyperlink ``[foo](foo.ipynb#bar)`` will be
+converted to the latex reference ``Section \ref{bar}`` and when compiled to PDF it will
+be rendered as something like ``Section 2.1``.
+
+Latex formatting
+----------------
+bookbook uses nbconvert under the hood.  Custom formatting of latex output
+can be accomplished by using a template, in the same way as would be done
+using nbconvert by itself.  See [the nbconvert
```
Comment (willingc) [inline README.rst]: Looks like you switched to markdown style links (instead of rst) for the remainder of this file.

**Review by willingc — COMMENTED** [review]
@ketch Overall, the additions are very helpful. One small comment on links in md instead of rst

**Comment by ketch** [comment]
Thanks @willingc!  That's  what happens when I try to write about markdown links in a RST document...They should all be correct now.

**Review by willingc — APPROVED** [review]
Thanks @ketch. @takluyver, LGTM.

**Comment by takluyver** [comment]
Thanks @willingc for the review :-)

**Pull request merged**